src/levanter/data/loader.py

```diff
@@ -172,8 +173,11 @@ def __init__(
         self.mesh = mesh
         self.Batch = Batch
 
-        process_data_pos = override_process_data_pos or levanter.mesh.process_mesh_position(mesh)[0]
-        num_data_process_groups = override_process_data_groups or levanter.mesh.process_mesh_size(mesh)[0]
+        process_mesh_map = process_mesh_mapping(self.mesh)
+        local_devices_map = local_devices_mapping(self.mesh)
+        logger.info(process_mesh_map, local_devices_map)
```

Comment:
change to debug

Reply:
do you mean `process_mesh_map[jax.process_index()] = override_process_data_pos`?
I'm not sure how to use `override_process_data_groups` for these mappings.